Assignment2/alejandrofdezmerino-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,42 @@
+{
+"@context": {
+	"vocab": "http://example.com/ontology/", 
+	"name": "vocab:name",
+	"hometown": "vocab:hometown",
+	"age": "vocab:age",
+	"almaMater": "vocab:almaMater",
+	"parents": "vocab:parents",
+	"subjects": "vocab:subjects",
+	"teachers": "vocab:teachers",
+	"description": "vocab:description"
+},
+	"@id": "http://www.example.com/ontology/myself",  
+	"name": "Alejandro Fernandez Merino",
+	"hometown": "Madrid",
+	"age": "23",
+	"almaMater": "Universidad Politecnica de Madrid",
+	"parents": "Africa y Julian",
+	"subjects":	{
```

Comment:
Define it as an array [ ]
Then you don't need to define "subject1": , "subject2": ,etc. Delete these names and leave only the name, teachers, description values inside curly braces.